fixtures/workers-with-assets-and-service-bindings/workerWS/src/index.ts

```diff
@@ -0,0 +1,20 @@
+import { WorkerEntrypoint } from "cloudflare:workers";
+
+export default class extends WorkerEntrypoint {
+	fetch(req: Request) {
+		if (!req.headers.get("Upgrade")?.includes("websocket")) {
+			return new Response("Hello from worker-ws fetch()");
+		}
+
+		const [server, client] = Object.values(new WebSocketPair());
+
+		server.accept();
```

Comment:
Should this go after you have setup the event listener?

Reply:
I guess it doesn't matter since the response hasn't been returned to the client yet.

Reply:
Is there a problem with this worker dying before the websocket ping-pong completes?
Do we need a `waitUntil` here?

Reply:
no idea tbh. is this something we need to worry about in the context of running the fixture tests? Feels like not, but maybe I'm wrong